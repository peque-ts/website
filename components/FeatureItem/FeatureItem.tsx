import React from 'react';

export const FeatureItem: React.VFC = () => {
  return <p>FeatureItem</p>;
};
