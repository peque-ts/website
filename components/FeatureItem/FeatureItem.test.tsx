import { render } from '@testing-library/react';

import { FeatureItem } from './FeatureItem';

describe('FeatureItem', () => {
  it('should render', () => {
    const { asFragment } = render(<FeatureItem />);

    expect(asFragment()).toMatchSnapshot();
  });
});
