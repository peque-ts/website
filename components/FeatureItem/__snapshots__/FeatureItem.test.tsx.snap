// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeatureItem should render 1`] = `
<DocumentFragment>
  <p>
    FeatureItem
  </p>
</DocumentFragment>
`;
